webapp/app/dashboards/routes.py

```diff
@@ -59,6 +66,37 @@
 CONST_SFP = 2.39  # specific fan power
 CONST_VTOT = 20337.0  # total volume – m3
 
+DEFAULT_SENSOR_TYPE = 3  # 3 is for Aranet T&RH
```

Comment:
I think this is fine, but does rely on us having added sensor types to the database in the same order in cropapp and cropapptest - it would be a bit clunky to look up the sensor id from the sensor type name, but would it be feasible to use the name everywhere in place of the id?